plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -376,7 +377,7 @@ func BuildDimensions(mTags map[string]string) []*cloudwatch.Dimension {
 		}
 
 		value := mTags[k]
-		if value == "" {
+		if dimensionExcluded[k] || value == "" {
```

Comment:
using struct{} here would be something like
```suggestion
		if _, ok := dimensionExcluded[k]; !ok || value == "" {
```